tests/TestClassFieldsCollectionConfiguratorForm.php

```diff
@@ -74,6 +74,27 @@ public function testConfigureForInterestForm()
 		}
 	}
 
+	/**
+	 * @throws UnknownFieldException
+	 */
+	public function testConfigureForInterestFormWithPassingKritBemerkungOeffentlichField()
+	{
+		$pFieldsCollection = $this->buildFieldsCollectionWithKritBemerkungOeffentlichField();
+		$pSubject = new FieldsCollectionConfiguratorForm;
+		$pFieldsCollectionNew = $pSubject->configureForInterestForm($pFieldsCollection);
+		$this->assertCount(1, $pFieldsCollectionNew->getAllFields());
+		foreach ($pFieldsCollectionNew->getAllFields() as $pField) {
+			$pFieldOriginal = $pFieldsCollection->getFieldByModuleAndName
+			($pField->getModule(), $pField->getName());
+			$this->assertNotSame($pFieldOriginal, $pField);
+			$this->assertThat($pField, $this->callback(function(Field $pField) {
+				return
+					($pField->getName() === 'objekttyp' && $pField->getType() === FieldTypes::FIELD_TYPE_MULTISELECT) ||
```

Comment:
I'm missing the check for `$pField->getName() !== 'krit_bemerkung_oeffentlich'` here. Other than that it looks pretty good

Reply:
Ok, I updated code, please check help me, thanks